test/e2e/user_certs_test.go

```diff
@@ -0,0 +1,162 @@
+package e2e
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"net/url"
+	"strings"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/util/cert"
+
+	configv1 "github.com/openshift/api/config/v1"
+	configclient "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	operatorclient "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
+
+	test "github.com/openshift/cluster-kube-apiserver-operator/test/library"
+)
+
+func TestNamedCertificates(t *testing.T) {
+
+	kubeConfig, err := test.NewClientConfigForTest()
+	test.FatalIfErr(t, err)
+	kubeClient, err := clientcorev1.NewForConfig(kubeConfig)
+	test.FatalIfErr(t, err)
+	configClient, err := configclient.NewForConfig(kubeConfig)
+	test.FatalIfErr(t, err)
+	operatorClient, err := operatorclient.NewForConfig(kubeConfig)
+	test.FatalIfErr(t, err)
+
+	initialConfigGeneration := test.GetKubeAPIServerOperatorConfigGeneration(t, operatorClient)
+
+	t.Log("Given the kube-apiserver-operator is available.")
+	test.WaitForKubeAPIServerClusterOperatorAvailableNotProgressingNotFailing(t, configClient)
+
+	t.Log("And some new tls Secrets have been created in the openshift-config namespace.")
+	rootCA := test.NewCertificateAuthorityCertificate(t, nil)
```

Comment:
the log does not match: this does not create anything in the openshift-config namespace